test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -1886,6 +1876,72 @@ context('action sidebar', function() {
       .and('contain', 'You are not able to change settings on this action.');
   });
 
+  specify('action with work:authored:delete permission', function() {
+    cy
+      .routesForPatientAction()
+      .routeCurrentClinician(fx => {
+        fx.data.relationships.role = { data: { id: '77777' } };
+        fx.data.relationships.team = { data: { id: '11111', type: 'teams' } };
+
+        return fx;
+      })
+      .routePatient(fx => {
+        fx.data.id = '1';
+
+        return fx;
+      })
+      .routeAction(fx => {
+        fx.data = {
+          id: '1',
+          attributes: {
+            name: 'Owned by the team',
```

Comment:
Should we name this `Authored by current user` or something like that? Since we're testing the authorship, not the ownership of the action.

Similar comment on the `'flow with work:authored:delete permission'` test as well.